internal/parser.go

```diff
@@ -208,7 +205,7 @@ func (p *Parser) replaceEnvironmentVariables(re *regexp.Regexp, str *string) {
 }
 
 func (p *Parser) expandFilePaths(file string) ([]string, error) {
-	filePaths := []string{}
+	var filePaths []string
```

Comment:
The variable initialization to an empty array is deliberate, so that I don't have to check against nil when iterating over it.

Reply:
ok ok, i'll restore this